cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/CaseManager.cs

```diff
@@ -2519,7 +2564,19 @@ private CaseSearchReply MapCases(IEnumerable<incident> cases)
                             .Select(f => new Flag
                             {
                                 Id = f.dfp_FlagId?.dfp_id,
-                                Description = f.dfp_FlagId?.dfp_description
+                                Description = f.dfp_FlagId?.dfp_description,
+                                FormId = f.dfp_FlagId?.dfp_formid
+                            }).ToArray(),
+                        MedicalConditions = c.dfp_incident_dfp_knownmedicalcondition
```

Comment:
note to self: there are 4 methods that use MapCases but none seem to be called by any of our apps, just integration tests. Which makes me wonder, do we use this change? Add a breakpoint here and see if this gets called but I think we can safely delete this new code to an Obsolete method